packages/carbon-graphs/src/js/helpers/axis.js

```diff
@@ -1122,6 +1139,13 @@ const getXAxisHeight = (config) => {
   const dummy = d3.select('body').append('div');
   const svg = dummy.append('svg');
   const group = svg.append('g').call(axis);
+  if (config.axis.x.ticks && config.axis.x.ticks.tickLabelsRotation === -45) {
```

Comment:
We shouldn't hardcode -45
```suggestion
  if (config.axis.x.ticks && config.axis.x.ticks.tickLabelsRotation !== 0) {
```